webui/react/src/hooks/useModal/UserSettings/useModalCreateUser.tsx

```diff
@@ -93,7 +93,8 @@ const useModalCreateUser = ({ onClose }: ModalProps): ModalHooks => {
   const modalOpen = useCallback(() => {
     openOrUpdate({
       closable: true,
-      content: <ModalForm branding={info.branding} form={form} />,
+      // passing a default brandind due to changes on the initial state
+      content: <ModalForm branding={info.branding || 'determined'} form={form} />,
```

Comment:
**suggestion:** use `BrandingType.Determined` instead of the string `determined`